www/include/reporting/dashboard/template/viewServicesGroupLog.ihtml

```diff
@@ -158,6 +158,12 @@
 			</tr>	
 </table>
 
+{literal}
+<script>
+	formatDateMoment();
+</script>
+{/literal}
+
```

Comment:
Maybe, Using script and literal tags in duplicate ?

Reply:
I removed the
</script>{/literal}{literal]<script>